<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectToSshWidget</class>
 <widget class="QWidget" name="ConnectToSshWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Remote profiling (SSH)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="contentContainer" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>204</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="hostname">
             <property name="placeholderText">
              <string>Hostname (IP address)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="port">
             <property name="placeholderText">
              <string>Port (default 22)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="user">
             <property name="placeholderText">
              <string>User</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="knownHostsPath">
             <property name="placeholderText">
              <string>Path to known_hosts file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="keyPath">
             <property name="placeholderText">
              <string>Path to private key file</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QRadioButton" name="signedDeploymentButton">
             <property name="text">
              <string>Signed Deployment</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="noDeploymentButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>OrbitService started manually</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sudoButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>Start OrbitService with sudo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sudoPassword">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>sudo password</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="connectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <widget class="orbit_session_setup::OverlayWidget" name="overlay" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>orbit_session_setup::OverlayWidget</class>
   <extends>QWidget</extends>
   <header>SessionSetup/OverlayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
